index.css

```diff
@@ -0,0 +1,92 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+* a{
+    text-decoration: none;
+}
+
+.index{
+    width: 100%;
+    font-family: 'Montserrat', sans-serif;
+    
+}
+
+.header{
+    width: 100%;
+    background-color: rgb(35, 39, 42);
+    color: #f3f4f6;
+    padding: 2rem 0rem;
+    text-align: center;
+    font-size: x-large;
+}
+
+.header h1{
+    display: inline-block; 
+}
+
+.main{
+    margin-top: 100px;
+    width: 100%;
+    padding: 2rem 0rem;
+    text-align: center;
+    background-color: #f3f4f6;
+    color: rgb(35, 39, 42);
+}
+
+.logo{
+    color: rgb(41, 189, 181);
+}
+
+.btn{
+    padding: 1rem 2rem;
+    border-radius: 3px;
+    border: 0;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 14px 28px, rgba(0, 0, 0, 0.22) 0px 10px 10px;
+    cursor: pointer;
+}
+
+.btn-primary{
+    background-color: rgb(35, 39, 42);
+    color: #f3f4f6;
+}
+
+.link{
+    width: 70%;
+    margin: auto;
+    color: #f3f4f6;
```

Comment:
I have noticed that you have been using the #f4f4f6 at multiple places, you have created a css variable for this hex code in colors.css named as offwhite-two but you haven't used here. Hence, I suggest you to import that file here and use that.